lib/streamlit/state/session_state.py

```diff
@@ -79,8 +73,8 @@ class Value:
 @attr.s(auto_attribs=True, slots=True, frozen=True)
 class WidgetMetadata:
     id: str
-    deserializer: WidgetDeserializer
-    serializer: WidgetSerializer
+    deserializer: WidgetDeserializer = attr.ib(repr=False)
```

Comment:
the others happen automatically because `auto_attribs` is set to true in the decorator. but these two I use it explicitly so I can have them removed from the generated repr